### PR TITLE
Refactor/styled jsx everywhere

### PR DESCRIPTION
**Pull request by HendrikThePendric**
I've removed all css files, and inline styles and replaced everything by styled-jsx.

work

### CODE REVIEW

File: src/components/OrgUnitsTreeWithExtraOptions.js [hunk 1]
```diff
@@ -32,14 +34,31 @@ export const OrgUnitsTreeWithExtraOptions = props => (
             </span>
             <div
                 id="extra-options"
-                style={getExtraOptionsStyle(props.showOptions)}
+                className={getExtraOptionsClassName(props.showOptions)}
```
Comment (Mohammer5) [inline src/components/OrgUnitsTreeWithExtraOptions.js]: I'm really curious how this works.. I just tested it and it works, but I still don't understand how it's done technically

File: src/components/TabularReport/ReportTable.js [hunk 2]
```diff
@@ -21,18 +21,43 @@ const ReportTable = ({ content: { title, headers, rows } }) => (
...
                 {rows.map((cells, index) => (
                     <Row cells={cells} key={`r-${cells[0]}-${index}`} />
                 ))}
             </tbody>
         </table>
+        <style jsx>{`
+            table {
+                border-collapse: collapse;
+            }
+            thead tr {
```
Comment (Mohammer5) [inline src/components/TabularReport/ReportTable.js]: There's no need for using `thead` in this selector, the `tr`s rendered by `Row` shouldn't be affected

File: src/components/TabularReport/ReportTable.js [hunk 3]
```diff
@@ -21,18 +21,43 @@ const ReportTable = ({ content: { title, headers, rows } }) => (
...
+            tbody :global(tr) {
+                border-bottom: 1px solid #e0e0e0;
+            }
+            th,
+            tbody :global(td) {
+                border: none;
+                font-size: 14px;
+                padding: 16px 16px;
+            }
+            table th {
```
Comment (Mohammer5) [inline src/components/TabularReport/ReportTable.js]: same as above

File: src/components/OrgUnitsTreeWithExtraOptions.js (hunk 1 above)
Comment (HendrikThePendric) [reply src/components/OrgUnitsTreeWithExtraOptions.js]: It just returns the string "show-options" or "hide-options". So it's . dynamic className

**Comment by HendrikThePendric** [comment]
Keeping this branch alive for retrospective review by @varl

**Pull request merged**